### PR TITLE
Technical/init linters

### PR DESCRIPTION
**Pull request by DandyCV**
Used gems:
```
gem 'bullet'
gem 'bundler-audit'
gem 'brakeman'
gem 'reek'
gem 'fasterer'
gem 'overcommit'
gem 'rails_best_practices'
gem 'rubocop'
gem 'rubocop-performance'
gem 'rubocop-rails'
gem 'rubocop-rspec'
```

### CODE REVIEW

File: .overcommit.yml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+  MergeConflicts:
+    enabled: true
+    description: 'Check for merge conflicts'
+    
+  YamlSyntax:
+    enabled: true
+    description: 'Check YAML syntax'
+    
+    BundleCheck:
```
Comment (bestwebua) [inline .overcommit.yml]: Please fix padding here

File: config/application.rb [hunk 2]
```diff
@@ -20,6 +20,7 @@
 Bundler.require(*Rails.groups)
 
 module PlannerApi
+  # Initialize Planner application
```
Comment (bestwebua) [inline config/application.rb]: You can disable this stylecop

File: config/environments/development.rb [hunk 3]
```diff
@@ -1,6 +1,16 @@
 require "active_support/core_ext/integer/time"
 
 Rails.application.configure do
+  config.after_initialize do
+    Bullet.enable        = true
```
Comment (bestwebua) [inline config/environments/development.rb]: Please remove bullet from develop env

File: config/environments/test.rb [hunk 4]
```diff
@@ -6,6 +6,12 @@
 # and recreated between test runs. Don't rely on the data there!
 
 Rails.application.configure do
+  config.after_initialize do
+    Bullet.enable        = true
```
Comment (bestwebua) [inline config/environments/test.rb]: Please move it to initializers: `config/initializers/bullet.rb` with next context:

```ruby
if defined?(Bullet)
  Rails.application.configure do
    config.after_initialize do
      Bullet.enable = true

      Bullet.bullet_logger = true
      Bullet.console = true
      Bullet.raise = Rails.env.test?
    end
  end
end
```

File: Gemfile [hunk 5]
```diff
@@ -17,12 +19,38 @@ gem 'puma', '~> 5.0'
...
+  gem 'rubocop-rspec', '~> 2.3.0', require: false
 end
 
 group :development do
   gem 'listen', '~> 3.3'
 end
 
+group :test do
+  # Code coverage for Ruby with a powerful configuration library and automatic merging of coverage across test suites
+  gem 'simplecov', require: false
```
Comment (bestwebua) [inline Gemfile]: Init simplecov after or with rspec env.

**Review by bestwebua — CHANGES_REQUESTED** [review]

File: .overcommit.yml (hunk 1 above)
Comment (DandyCV) [reply .overcommit.yml]: fixed

File: config/initializers/bullet.rb [hunk 6]
```diff
@@ -0,0 +1,11 @@
...
+    Rails.application.configure do
+      config.after_initialize do
+        Bullet.enable = true
+  
+        Bullet.bullet_logger = true
+        Bullet.console = true
+        Bullet.raise = Rails.env.test?
+      end
+    end
+  end
```
Comment (bestwebua) [inline config/initializers/bullet.rb]: Please add ending empty line.

**Review by bestwebua — CHANGES_REQUESTED** [review]

File: config/environments/development.rb (hunk 3 above)
Comment (DandyCV) [reply config/environments/development.rb]: removed

File: config/environments/test.rb (hunk 4 above)
Comment (DandyCV) [reply config/environments/test.rb]: Added this file with config

File: Gemfile (hunk 5 above)
Comment (DandyCV) [reply Gemfile]: commented this gem

File: config/initializers/bullet.rb (hunk 6 above)
Comment (DandyCV) [reply config/initializers/bullet.rb]: done!

File: config/initializers/bullet.rb [hunk 7]
```diff
@@ -0,0 +1,12 @@
...
+
```
Comment (bestwebua) [inline config/initializers/bullet.rb]: Just empty line.

**Review by bestwebua — CHANGES_REQUESTED** [review]

File: config/initializers/bullet.rb (hunk 7 above)
Comment (DandyCV) [reply config/initializers/bullet.rb]: fixed

File: config/application.rb (hunk 2 above)
Comment (DandyCV) [reply config/application.rb]: Done!

File: config/environments/test.rb [hunk 8]
```diff
@@ -6,6 +6,12 @@
 # and recreated between test runs. Don't rely on the data there!
 
 Rails.application.configure do
+  config.after_initialize do
```
Comment (bestwebua) [inline config/environments/test.rb]: Why bullet configured in both places?

**Review by bestwebua — CHANGES_REQUESTED** [review]

**Review by bestwebua — APPROVED** [review]
👍

**Pull request merged**